### ISSUE TITLE
[request] Add media information

### ISSUE THREAD

**Issue by nimmy404** [nikitasius/Telegraher#158]
Add media information like file size on all media and stuff.

**Comment by nikitasius** [comment]
good idea. modern vanilla have it for many files, guess it could be nice to add it for more cases and make it permanent (kind of small labels on medias/files/etc)
